### ISSUE TITLE
Specify backup settings per location

### ISSUE THREAD

**Issue by VibeGAMESNL** [pterodactyl/panel#4411]
### Is there an existing feature request for this?

- [X] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

Currently if we enable S3 backup in the .env config file it will apply for all nodes/locations. But since we have locations all arround the world, we do not want to let a node in South Africa backup to a S3 endpoint in the US. Because thats extremely slow. In some cases we also want to use a local (ftp) backup for a specific location.

It would be nice to be able to choose a backup server/method per location.

### Describe the solution you'd like.

In the Admin panel under **Locations** there could be an option where we can choose between "**local backup**" or "**S3 backup**" for the nodes attached to that location. And optionally be able to specify the S3 backup settings for that location.

**Comment by wavy-cat** [comment]
I think it's better to choose a backup server every time in the form of creating a new backup. So that you can store backups of one server in different places at the same time. You can also choose not only between S3 storage, but also between local storage and remote storage.

**Comment by anondev-sudo** [comment]
Yeah. Also more backups options like external FTP (SFTP). Backup settings should be editable in admin area, not with env file.

**Comment by DaneEveritt** [comment]
In the interest of maintainer sanity and cleaning up an enormous backlog of stale feature requests we have moved to requiring all feature requests be discussed within [Discussions](https://github.com/orgs/pterodactyl/discussions/categories/feature-requests). Please consider re-opening this request there if you believe it is still relevant and worth consideration.

See https://github.com/pterodactyl/panel/issues/5491 for additional context.

Thanks.

<sub>_This comment is in the interest of issue tracker maintenance: it does not imply that maintainers are endorsing or dismissing your request._</sub>
